src/test/app/forms/validation/validators/IsLessThanOrEqualToSumOf.ts

```diff
@@ -0,0 +1,51 @@
+import { expect } from 'chai'
+import { IsLessThanOrEqualToSumOfConstraint } from 'forms/validation/validators/isLessThanOrEqualToSumOf'
+import { ValidationArguments } from 'class-validator'
+
+const DEFAULT_FIELD_TO_SUM_VALUE = 30
+const DEFAULT_VALUE = 200
+
+function validationArgs (fieldToSumValue: number, value: number): ValidationArguments {
+  return {
+    value: undefined,
+    targetName: undefined,
+    object: { fieldToSum: fieldToSumValue, relatedField: value },
+    property: undefined,
+    constraints: ['fieldToSum', 'relatedField']
```

Comment:
That is outdated now, in this context we could probably use empty array `[]`.

Also for the test we probably only need to populate constraint property:

```
function validationArgs (sum: number, lessThenAmount: number): ValidationArguments {
  return {
    constraints: [sum, lessThenAmount]
  } as ValidationArguments
}
```

Reply:
that's actually the names of the field in the dummy object  I created for validation

Reply:
Right, forgot the `object: {}` is actually an instance of class under validation. Ignore me then.